package.json

```diff
@@ -16,10 +16,10 @@
     "http-proxy": "1.13.2",
     "argparse": "1.0.6",
     "cli-interact": "0.1.9",
-    "throttle": "",
-    "ytdl-core": "",
-    "fluent-ffmpeg": "",
-    "through2": ""
+    "throttle": "1.0.3",
+    "ytdl-core": "0.11.2",
```

Comment:
Got caught out by not fixing dependencies and streaming broke. After messing about for a while I found that the latest version of ytdl-core was the cause. Not sure why yet, but downgrading it fixes it so I'm happy.